pkg/validate/common.go

```diff
@@ -110,6 +110,9 @@ func validateStringList(vf ValidatorFunc) ValidatorFunc {
 			return util.NewErrs(err)
 		}
 		var errs util.Errors
+		if val.(string) == "*" {
```

Comment:
This logic doesn't seem quite right. This function is intended as a wrapper to call a validator function vf, which validates each field. There should not be any field specific validation logic here. 
Rather you can create a validator function which accepts either an IP address list or "*" - if it's "*" return ok, or otherwise call this function.

Reply:
Updated.